lib/gds-components/notification-banner/NotificationBanner.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * GDS component: https://design-system.service.gov.uk/components/notification-banner/
+ */
+
+import { toDefaultLocalDateString } from "../..//utils/timetable";
```

Comment:
Looks like an extra `/` in the path (before "utils")

Reply:
done :)